src/components/Recipes/recipeCard/index.jsx

```diff
@@ -21,19 +21,19 @@ export const RecipeCard = ({
   const navigate = useNavigate();
 
   if (isLoading) return <Loading />;
-  
+
   return (
     <div
       onClick={(e) => {
-        if (!e.target.closest('.user-link')) {
+        if (!e.target.closest(".user-link")) {
           e.stopPropagation();
```

Comment:
After this pr please install prettier and linter to avoid those non wanted changing

Reply:
i have prettier